src/libraries/System.Numerics.Tensors/tests/ReadOnlyTensorSpanTests.cs

```diff
@@ -206,34 +204,34 @@ public static void ReadOnlyTensorSpanSystemArrayConstructorTests()
             Assert.Equal(94, spanInt[1, 0]);
             Assert.Equal(94, spanInt[1, 1]);
 
-            //Make sure it works with NIndex
-            spanInt = new ReadOnlyTensorSpan<int>(a, (NIndex[])[1, 1], [2, 2], [0, 0]);
-            Assert.Equal(2, spanInt.Rank);
-            Assert.Equal(2, spanInt.Lengths[0]);
-            Assert.Equal(2, spanInt.Lengths[1]);
-            Assert.Equal(94, spanInt[0, 0]);
-            Assert.Equal(94, spanInt[0, 1]);
-            Assert.Equal(94, spanInt[1, 0]);
-            Assert.Equal(94, spanInt[1, 1]);
-
-            //Make sure it works with NIndex
-            spanInt = new ReadOnlyTensorSpan<int>(a, (NIndex[])[^1, ^1], [2, 2], [0, 0]);
-            Assert.Equal(2, spanInt.Rank);
-            Assert.Equal(2, spanInt.Lengths[0]);
-            Assert.Equal(2, spanInt.Lengths[1]);
-            Assert.Equal(94, spanInt[0, 0]);
-            Assert.Equal(94, spanInt[0, 1]);
-            Assert.Equal(94, spanInt[1, 0]);
-            Assert.Equal(94, spanInt[1, 1]);
+            // // TODO: Make sure it works with NIndex
```

Comment:
Will these tests get done before merging?

Reply:
No, they would be for additional APIs that need to be exposed after the Build preview